source/Calamari.Tests/Fixtures/RequiresMinimumPython3VersionAttribute.cs

```diff
@@ -0,0 +1,69 @@
+using System;
+using System.Linq;
+using System.Text.RegularExpressions;
+using Calamari.Integration.Processes;
+using Calamari.Integration.Scripting.Python;
+using Calamari.Integration.ServiceMessages;
+using Calamari.Tests.Helpers;
+using NUnit.Framework;
+using NUnit.Framework.Interfaces;
+using Octopus.CoreUtilities;
+using Octostache;
+
+namespace Calamari.Tests.Fixtures
+{
+    public class RequiresMinimumPython3VersionAttribute : TestAttribute, ITestAction
+    {
+        const int Major = 3;
+        readonly int minor;
+
+        public RequiresMinimumPython3VersionAttribute(int minor)
+        {
+            this.minor = minor;
+        }
+
+        public void BeforeTest(ITest test)
+        {
+            var python3Version = GetPython3Version();
+            if (python3Version.None())
+            {
+                Assert.Ignore("Requires Python3 to be installed");
```

Comment:
I prefer `Assert.Inconclusive` for these kind of ones - they show up better for this scenario